lang/ocaml/Portfile

```diff
@@ -14,7 +14,7 @@ PortGroup           compiler_blacklist_versions 1.0
 name                ocaml
 epoch               1
 version             4.08.1
-revision            0
+revision            2
```

Comment:
The `revision` only needs to increased to 1.

Reply:
I am happy to make the change, but I need a little feedback as to what you prefer.
There are two conceptually distinct changes in this pull request that each require a revision increase (remove ncurses as a dependency and respect build_arch).
Would you prefer:
* The two distinct changes are merged so that only one revision increase is needed?
* Only one change gets a revision increase with a note that a revision bump is imminent?

Reply:
If you are pushing these at the same time, only a single revision increase is needed. If not, then you can increment twice.

Reply:
So you do not have a strong opinion about how the history should read?
Are you OK with the changes themselves?

Reply:
The `revision` is only about users getting updates when performing `port upgrade`. Yes, the changes look fine to me. However, I know very little about ocaml ports.